### DISCUSSION TITLE
Rate limiting headers

### DISCUSSION THREAD

**Discussion by zulufoxtrot** [Hyundai-Kia-Connect/hyundai_kia_connect_api#270] [General]
I was experimenting with requests (Kia EU) when I noticed these headers in the response:

```
('X-Ratelimit-Limit', '100')
('X-Ratelimit-Remaining', '100')
('X-Ratelimit-Reset', '1674045213')
```

Wondering if this data is reliable and if/how we could use it 🤔 

I've been doing requests all day, so I'm surprised to see the "remaining" counter at 100.
Also, I thought the limit was 200 not 100.
The last value is a timestamp that dates back to 4 days ago. Which is strange because I got rate limited as late as 2 days ago.

**Comment by cdnninja** [comment]
If they were accurate would be handy to expose to calling services.

**Comment by zulufoxtrot** [comment]
I did some more testing tonight and got rate limited in the middle.
Data remained unchanged before and after rate limiting was enforced:
Limit: 100
Remaining: 100
Reset: a timestamp from a couple days ago

I think that, right now, this data is not reliable.
